fontbe/src/head.rs

```diff
@@ -62,12 +61,56 @@ fn build_head(units_per_em: u16, loca_format: LocaFormat) -> Head {
     }
 }
 
+/// See:
+/// * <https://www.microsoft.com/typography/otspec/recom.htm>
+/// * <https://github.com/googlefonts/ufo2ft/blob/0d2688cd847d003b41104534d16973f72ef26c40/Lib/ufo2ft/outlineCompiler.py#L313-L326>
+fn apply_font_revision(head: &mut Head, major: i32, minor: u32) {
+    let major = major as f64;
+    let minor = if minor != 0 {
+        // Make minor the decimal part
+        let mut minor = minor as f64;
+        minor /= 10.0_f64.powi(minor.log10().ceil() as i32);
+        // Keep 3 decimal places per OTSpec recommendation
+        minor = (minor * 1000.0).round() / 1000.0;
+        minor
+    } else {
+        0.0
+    };
+
+    head.font_revision = Fixed::from_f64(major + minor);
+}
+
+fn apply_created_modified(
+    head: &mut Head,
+    created: Option<DateTime<Utc>>,
+    modified: Option<DateTime<Utc>>,
+) {
+    let now = current_timestamp();
+    head.created = LongDateTime::new(created.map(seconds_since_mac_epoch).unwrap_or(now));
+    head.modified = LongDateTime::new(modified.map(seconds_since_mac_epoch).unwrap_or(now));
+}
+
 impl Work<Context, Error> for HeadWork {
     /// Generate [head](https://learn.microsoft.com/en-us/typography/opentype/spec/head)
     fn exec(&self, context: &Context) -> Result<(), Error> {
         let static_metadata = context.ir.get_final_static_metadata();
         let loca_format = context.get_loca_format();
-        let head = build_head(static_metadata.units_per_em, *loca_format);
+        let mut head = init_head(
```

Comment:
since you're touching head, would be nice if you could also set the head.macStyle bits (for bold/italic, work similarly to OS/2 fsSelection),

https://github.com/googlefonts/ufo2ft/blob/0d2688cd847d003b41104534d16973f72ef26c40/Lib/ufo2ft/outlineCompiler.py#L346-L354

or maybe you can punt until #300